SteamProject/MySteam/Game.h

```diff
@@ -8,28 +8,34 @@ class FGame
     int ReleaseDay;
     int ReleaseMonth;
     int ReleaseYear;
-    
+
 public:
 
     FGame();
 
-    FGame(const std::string NewGameName, const std::string NewStudioName, const int NewReleaseDay, const int NewReleaseMonth, const int NewReleaseYear);
+    FGame(const std::string& NewGameName, const std::string& NewStudioName, const int& NewReleaseDay, const int& NewReleaseMonth, const int& NewReleaseYear);
+
+    FGame(const FGame &InGame);
+
+    ~FGame();
     
-    std::string GetGameName() const;
+    const std::string& GetGameName() const;
 
-    std::string GetStudioName() const;
+    const std::string& GetStudioName() const;
     
-    std::string GetReleaseDate() const;
+    const std::string GetReleaseDate() const;
```

Comment:
Keep in mind that functions should never return const types. They can return const references or const pointers to other types, but they shouldn't return const types directly.

https://app.tettra.co/teams/tg-academy/pages/coding-standards#header-erfjj-const-correctness-guidelines